frontend/pages/hubs/[hubUrl].js

```diff
@@ -305,6 +329,18 @@ export default function Hub({
               hubUrl={hubUrl}
             />
           </div>
+          {!user && isLocationHub && (
+            <SignUpPromptDialog
+              buttonText={texts.join + "!"}
+              image={"/images/sign_up_prompt_image.jpg"}
+              infoTextOne={texts.receive_regular_updates_about_projects_topics}
+              infoTextTwo={texts.send_messages_comments_and_more}
+              onClose={handleCloseSignUpPrompt}
+              open={showSignUpPrompt}
+              subTitle={texts.sign_up_today_and_help_make_climate_neutral}
+              title={texts.change_the_world}
```

Comment:
Nit: do you mind sorting these props? If you use VS Code this should be really easy with `cmd+shift+p` and type 'alphabetize'.